spk/tcl/src/installer.sh

```diff
@@ -33,10 +35,13 @@ preuninst ()
 postuninst ()
 {
 	# Remove symlinks
+	rm -f /usr/local/${PACKAGE}
```

Comment:
My bad, scratch that comment... We do remove the install dir via postuninst, but it looks like this:
```
# Remove link
rm -f ${INSTALL_DIR}
```